XamlControlsGallery/ControlPages/ListViewPage.xaml.cs

```diff
@@ -56,6 +85,241 @@ private void SelectionModeComboBox_SelectionChanged(object sender, SelectionChan
+                    {
+                        contacts2.Remove(contact);
+                        break;
+                    }
+                }
+            }
+
+            e.AcceptedOperation = DataPackageOperation.Move;
+            def.Complete();
+        }
+
+        private void Target_DragItemsStarting(object sender, DragItemsStartingEventArgs e)
+        {
+            if (e.Items.Count == 1)
+            {
+                // Prepare ListViewItem to be moved
+                Contact tmp = (Contact)e.Items[0];
+
+                e.Data.SetText(tmp.FirstName + " " + tmp.LastName + " " + tmp.Company);
+                e.Data.RequestedOperation = DataPackageOperation.Move;
+            }
+        }
+
+        private void Target_DragEnter(object sender, DragEventArgs e)
+        {
+            // We don't want to show the Move icon
+            e.DragUIOverride.IsGlyphVisible = false;
+        }
+
+        //===================================================================================================================
+        // Grouped Headers Example
+        //===================================================================================================================
+        private void ToggleSwitch_Toggled(object sender, RoutedEventArgs e)
+        {
+            if (StickySwitch != null)
+            {
+                if (StickySwitch.IsOn == true)
+                {
+                    stackPanelObj.AreStickyGroupHeadersEnabled = true;
+                }
+                else
+                {
+                    stackPanelObj.AreStickyGroupHeadersEnabled = false;
+                }
+            }
+        }
+
+        private void StackPanel_loaded(object sender, RoutedEventArgs e)
+        {
+            stackPanelObj = sender as ItemsStackPanel;
+        }
+
+        //===================================================================================================================
+        // Inverted List Example
+        //===================================================================================================================
+
+        private void AddItemToEnd()
+        {
+
+            InvertedListView.Items.Add(
+                new Message("Message " + ++messageNumber, DateTime.Now.ToString(), "Right")
+                );
+        }
+
+        private void MessageRecieved(object sender, RoutedEventArgs e)
+        {
+            InvertedListView.Items.Add(
+                new Message("Message " + ++messageNumber, DateTime.Now.ToString(), "Left")
+                );
+        }
+    }
+
+    public class Message
+    {
+        public string MsgText { get; private set; }
+        public string MsgDateTime { get; private set; }
+        public string MsgAlignment { get; set; }
+        public SolidColorBrush BgColor { get; set; }
+        public Message(string text, string datetime, string align)
```

Comment:
I would expect "dateTime" to be a DateTime type. Similarly, I'd expect "align" to be an Alignment type. 

Consider passing in a DateTime object and turning it into a string inside this method. 

For alignment: consider adding an enum to this class to help force the alignment. Passing around strings is generally unsafe when you expect certain values.